python_modules/dagster/dagster/_core/storage/event_log/sqlite/sqlite_event_log.py

```diff
@@ -184,6 +184,7 @@ def _initdb(self, engine: Engine) -> None:
                     or "table event_logs already exists" in err_msg
                     or "table asset_event_tags already exists" in err_msg
                     or "table alembic_version already exists" in err_msg
+                    or "table dynamic_partitions already exists" in err_msg
```

Comment:
<i class='graphite__hidden'>[Re: lines 182 to 187]<br/></i>

i guess it would need to be regex to handle this better but we should maybe try to forward proof this better since its not great to have to come back and update this <span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/12367?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
Swapped this with regex!